app/views/games/show.html.erb

```diff
@@ -2,10 +2,80 @@
 
 <div class="booyah-box col-xs-10 col-xs-offset-1">
   <div class="chessboard">
-    <% (0..7).each do |row| %>
+    <% (7.downto(0)).each do |row| %>
```

Comment:
While this makes sense from a chess perspective, my guess is it will make other parts of the app hard to work with since the array indexes for the rows will start at 0.  My suggestion would be to just go from 0 at the top to 7 at the bottom here, and if we need to reverse it on display we can do that.